[2025-01-31T23:56:44.787+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T23:56:44.831+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-01-30T23:57:22.460856+00:00 [queued]>
[2025-01-31T23:56:44.851+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-01-30T23:57:22.460856+00:00 [queued]>
[2025-01-31T23:56:44.851+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 3
[2025-01-31T23:56:44.874+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): assign_permissions_dag> on 2025-01-30 23:57:22.460856+00:00
[2025-01-31T23:56:44.880+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=486) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T23:56:44.882+0000] {standard_task_runner.py:72} INFO - Started process 487 to run task
[2025-01-31T23:56:44.883+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'manage_***_roles', 'assign_permissions_dag', 'manual__2025-01-30T23:57:22.460856+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpv3haevmz']
[2025-01-31T23:56:44.884+0000] {standard_task_runner.py:105} INFO - Job 90: Subtask assign_permissions_dag
[2025-01-31T23:56:44.988+0000] {task_command.py:467} INFO - Running <TaskInstance: manage_airflow_roles.assign_permissions_dag manual__2025-01-30T23:57:22.460856+00:00 [running]> on host dab0f5b0cfba
[2025-01-31T23:56:45.243+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='manage_***_roles' AIRFLOW_CTX_TASK_ID='assign_permissions_dag' AIRFLOW_CTX_EXECUTION_DATE='2025-01-30T23:57:22.460856+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-30T23:57:22.460856+00:00'
[2025-01-31T23:56:45.245+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T23:56:45.251+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-31T23:56:45.277+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T23:56:45.279+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=manage_***_roles, task_id=assign_permissions_dag, run_id=manual__2025-01-30T23:57:22.460856+00:00, execution_date=20250130T235722, start_date=20250131T235644, end_date=20250131T235645
[2025-01-31T23:56:45.338+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T23:56:45.394+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-31T23:56:45.397+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
